desci-server/prisma/schema.prisma

```diff
@@ -806,6 +807,16 @@ model OrcidPutCodes {
   @@unique([orcid, uuid, reference])
 }
 
+model DoiRecord {
+  id        Int      @id @default(autoincrement())
+  doi       String   @unique
+  dpid      String   @unique
+  uuid      String   @unique
```

Comment:
IMO we shouldn't rely on uuid since the whole dot thing is tough to work with, we should consider making uuid in their current form deprecated and use either nodeId or uuid in hex format

Reply:
I used uuid here because there's no direct link between Users -> Dpid in the database. To get that we'd have to make calls to the subgraph to retrieve the uuid which is not efficient.

Reply:
If we change anything about the UUID's we should just drop the dot(s) as it serves zero purpose. It's incorrectly formatted, unnecessary padding :roll_eyes: